packages/runtime/src/next/client.ts

```diff
@@ -39,12 +59,12 @@ export type MakeswiftPageDocument = {
     isIndexingBlocked: boolean | null
   }
 }
+type MakeswiftResources = ElementTreeResources & NonElementTreeResources
```

Comment:
This distinction feels a bit artificial, but it has two real world effects:
1. `NonElementTreeResources` come from the `/document` endpoint, which we could clean up in the future, but we have enough for now.
2. `ElementTreeResources`, "cache data", and `SerializedState` are all the same. We should probably consolidate that, or maybe bring all of the resources into the Redux store for simplicity.